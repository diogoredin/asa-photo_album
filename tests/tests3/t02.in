100 196
51 74
33 18
98 24
52 14
18 35
86 46
42 47
54 3
99 45
51 75
45 70
47 3
50 16
64 92
54 40
95 82
31 94
63 83
81 48
77 87
31 96
32 26
92 57
5 76
26 13
41 94
39 8
43 77
80 97
84 19
61 53
60 14
9 45
89 10
30 90
31 62
82 83
78 36
87 73
94 2
90 59
42 10
46 13
29 61
25 98
7 11
12 33
27 56
52 38
19 40
11 51
5 83
56 2
88 49
20 65
13 23
14 68
14 45
93 41
51 58
56 81
64 40
8 58
89 55
34 20
58 96
30 10
67 74
9 95
93 94
2 92
38 97
99 89
18 22
99 20
83 24
74 54
31 28
46 12
67 64
9 88
2 60
10 20
6 66
82 6
85 79
72 68
66 18
73 17
80 92
64 14
13 45
68 34
97 78
23 79
59 93
34 39
86 49
30 27
75 4
3 88
50 5
57 37
37 21
49 25
48 14
27 68
12 76
46 22
100 99
91 72
87 68
47 75
75 3
29 98
76 31
4 90
13 44
89 46
97 20
76 13
22 5
95 59
11 94
16 52
52 31
32 69
98 14
96 1
67 73
64 22
97 42
79 74
22 50
85 16
81 28
44 37
69 28
31 75
35 30
48 5
49 57
84 29
36 45
47 31
89 67
62 69
14 83
44 80
28 79
86 34
53 23
44 8
55 43
44 74
15 7
26 91
65 92
28 15
33 71
50 35
72 53
55 42
22 11
66 55
70 63
52 42
67 34
21 71
89 32
89 23
71 78
72 92
1 10
81 85
80 85
46 45
9 74
73 71
24 19
89 50
17 86
56 54
74 65
91 50
72 47
72 67
89 39
22 51
100 69
51 63
38 42
60 84
64 9
94 53
49 58
