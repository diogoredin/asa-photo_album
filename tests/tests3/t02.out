100 99 89 32 26 91 72 67 64 9 95 82 6 66 55 43 77 87 73 17 86 46 12 33 18 22 50 35 30 27 56 81 48 5 76 13 44 80 85 16 52 38 97 42 47 31 62 69 28 15 7 11 51 75 4 90 59 93 41 94 2 60 84 29 61 53 23 79 74 54 3 88 49 25 98 14 68 34 39 8 58 96 1 10 20 65 92 57 37 21 71 78 36 45 70 63 83 24 19 40
